op-conductor/conductor/service_test.go

```diff
@@ -106,7 +106,7 @@ func (s *OpConductorTestSuite) SetupSuite() {
 	s.metrics = &metrics.NoopMetricsImpl{}
 	s.cfg = mockConfig(s.T())
 	s.version = "v0.0.1"
-	s.next = make(chan struct{}, 1)
+	s.next = make(chan struct{})
```

Comment:
This does not need to be a buffered channel, and in fact, a block channel would be better for synchronization